src/components/EditorPage/EditorPage.tsx

```diff
@@ -53,9 +53,12 @@ export default class EditorPage extends React.PureComponent<Props, State> {
   }
 
   componentDidUpdate(): void {
-    const { currentProject, claimedName, isFromClaimName, isLoading, isPreviewing, onOpenModal } = this.props
+    const { currentProject, claimedName, isFromClaimName, isLoading, isPreviewing, isScreenshotReady, onOpenModal, onTakeScreenshot } =
+      this.props
     const { isDeployModalOpened } = this.state
-    if (!(isLoading || isPreviewing) && currentProject && isFromClaimName && !isDeployModalOpened) {
+    if (!(isLoading || isPreviewing) && isScreenshotReady && currentProject && !currentProject.thumbnail) {
```

Comment:
Take a new screenshot if for any reason the first thumbnail wasn't successfully taken